node/src/Pos/Update/Download.hs

```diff
@@ -9,6 +9,7 @@ module Pos.Update.Download
 import           Universum
 
 import           Control.Lens            (views)
+import           Control.Monad.Catch     (handleAll)
```

Comment:
Don't use this module use [safe functions](https://www.stackage.org/haddock/lts-9.2/safe-exceptions-0.1.6.0/Control-Exception-Safe.html#v:handleAny).